<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowPrincipal</class>
 <widget class="QMainWindow" name="MainWindowPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>1434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application BDD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Database's avaible :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDatabase"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Table's of the chosen database :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Fields of the chosen tables :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidgetTable"/>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="listWidgetChamps"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>History of your SQL request :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetRequeteHistorique"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonChargerHistorique">
      <property name="text">
       <string>&amp;Load history</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEnregistrerHistorique">
      <property name="text">
       <string>&amp;Save history</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Your SQL request :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditRequete"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonVerificationRequete">
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditErreurRequete"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEnregistrerRequete">
      <property name="text">
       <string>Save SQL request</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Results of your SQL requests :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetResultatRequete"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExporterCSV">
      <property name="text">
       <string>Export the result on CSV format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichiers">
    <property name="title">
     <string>&amp;Files</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
   </widget>
   <addaction name="menu_Fichiers"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
